src/pages/search/search.styles.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: calc(100vh - 200px);
+`;
+
+export const Form = styled.form`
+  position: relative;
+  top: -40px;
+`;
+
+export const Input = styled.input`
+  border: 1px solid #d5d5d5;
+  color: black;
+  font-size: 14px;
+  height: 36px;
+  margin: 0px 10px;
+  outline: none;
+  padding-left: 16px;
+  width: 370px;
+`;
+
+export const SubredditLabel = styled.label`
+  color: #9e9e9e;
+  font-size: 18px;
+  position: relaive;
+  left: 5px;
+`;
+
+export const SearchButton = styled.button`
```

Comment:
Many of the button styles are probably duplicated here. A separate component might make sense.